src/core/token/TokenManager.ttslua

```diff
@@ -149,8 +149,12 @@ do
   --     spawned state object rather than spawning multiple tokens
   ---@param shiftDown An offset for the z-value of this group of tokens
   TokenManager.spawnTokenGroup = function(card, tokenType, tokenCount, shiftDown)
+    local optionPanel = Global.getTable("optionPanel")
```

Comment:
Comment for future, but we really need to convert this to an API and have a method to get each option value.  Other objects needing to know the keys that are being used (and hardcoding them) is going to be fragile.

Reply:
True, that would make options that affect other objects / script easier to implement too